src/test/java/com/wikia/webdriver/testcases/flagTests/flagTest.java

```diff
@@ -0,0 +1,47 @@
+package com.wikia.webdriver.testcases.flagtests;
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+/**
+ * Created by Mariusz on 2015-06-28.
+ */
+public class FlagTest extends NewTestTemplate {
+
+    Credentials credentials = config.getCredentials();
+
+    @Test(
+            groups = {"addFlag_01"})
+    public void addFlag_01() {
+        List list = new ArrayList();
+        WikiBasePageObject base = new WikiBasePageObject(driver);
+        base.logInCookie(credentials.userNameStaff, credentials.passwordStaff,URLsContent.EXTERNAL_RUNESCAPE);
+        base.openArticleByName(URLsContent.EXTERNAL_RUNESCAPE,URLsContent.FlagTest);
+        base.openFlagModal();
+        List<WebElement> inputs = base.getListFlags();
+        //take flags ID which are enabled on current wiki
+        //we need to take id because next we check if flag is enabled on page and this need ID flag
+        for (WebElement input : inputs){
+            if (!driver.findElement(By.id(input.getAttribute("id"))).isSelected())
+                driver.findElement(By.id(input.getAttribute("id"))).click();
```

Comment:
I assume you want to check if input element is not selected and if so click it. You don't need to find this element by it's id, you can just check if this element is not selected and tap it:
```
if (!input.isSelected()) {
        input.click()
}
```